transformer/kubernetes/apiresource/argocdapplication.go

```diff
@@ -70,6 +71,14 @@ func (*ArgoCDApplication) createNewResource(irApplication irtypes.Application, t
 		clusterServer = deployToSameCluster
 	}
 	appGVK := v1alpha1.ApplicationSchemaGroupVersionKind
+	ques := qaengine.FetchStringAnswer(
+		common.ConfigTransformersKubernetesArgoCDNamespaceKey,
+		"Enter destination namespace for argo cd pipeline",
+		[]string{"If this is not relevant to you then give an empty string to remove it from the YAML."},
```

Comment:
Can we also update the context to `[]string{If Argo CD pipeline is not relevant to you, then leave empty to use the default value for it.}`